front-end/components/views/Auth/SignUpView.jsx

```diff
@@ -74,14 +81,14 @@ class SignUpView extends React.Component {
 
         <div style={{ height: '10vw' }}></div>
 
-        <CustomButton
-          buttonClass='rainbow-button'
-          buttonTextClass='large-white'
-          buttonText='Login'
-          buttonCallback={this.signUpButtonClick}>
-        </CustomButton>
-        <h2 style={{ textAlign: 'center', fontFamily: 'var(--theme-font)' }}>
-          Create Account</h2>
+        <Link to="/preview">
+          <CustomButton
+            buttonClass='rainbow-button'
```

Comment:
This does work, but it is a bit unusual. Usually, you do not want to be setting the class of a subview from the parent.